library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -1697,13 +1697,15 @@
     <string name="settings_push_rules_no_rules">No push rules defined</string>
     <string name="settings_push_gateway_no_pushers">No registered push gateways</string>
 
-    <string name="push_gateway_item_app_id">app_id:</string>
-    <string name="push_gateway_item_push_key">push_key:</string>
-    <string name="push_gateway_item_app_display_name">app_display_name:</string>
-    <string name="push_gateway_item_device_name">session_name:</string>
+    <string name="push_gateway_item_app_id">App ID:</string>
+    <string name="push_gateway_item_push_key">Push Key:</string>
+    <string name="push_gateway_item_app_display_name">App Display Name:</string>
+    <string name="push_gateway_item_device_name">Device Display Name:</string>
```

Comment:
Maybe we should stick to `Session Display Name:` and `Session ID:` on the UI side to keep consistency with other sreens related to sessions?